docs/runbooks/CDIDataImportCronOutdated.md

```diff
@@ -0,0 +1,72 @@
+# CDIDataImportCronOutdated
+
+## Meaning
+
+DataImportCron is in charge of recurring polling of latest version disk images as PVCs, commonly into a 'golden image' namespace;   
+These are PVCs that always get updated with latest version,  
+serving as a trustworthy clone source for created VMs (created from latest image of OS).  
+
+This alert fires when a DataImportCron fails to create a corresponding PVC or keep its corresponding PVC updated (new `:latest` exists, we can't update to it).
+
+**Note**:  
+In the golden-image use case, `latest` is simply the latest OS version of the distribution.  
+In the non-golden-image case, as described above, we are merely referring to the latest hash of the image that is available.
+
+## Impact
+
+VMs are created from outdated disk images *or* VMs fail to start because there is no source PVC to clone from.
+
+## Diagnosis
+
+### Distinguish between golden images and regular crons
+DataImportCrons are widely used for golden images and likely serve as a source to create VM disks from, and thus it is vital for them to be up to date.
+- Find the erronous DataImportCron's namespace & name:
+	```bash
+	kubectl get dataimportcron -A -o json | jq -r '.items[] | select(.status.conditions[] | select(.type == "UpToDate" and .status == "False")) | .metadata.namespace + "/" + .metadata.name'
+	```
+	Output is returned as namespace/name - golden image crons will reside in the `openshift-virtualization-os-images` namespace.
```

Comment:
What about non-openshift clusters? This doc is for both upstream and downstream.

Reply:
This means they are not golden images (the section is just about identifying)